debian/marquee/nginx/sites/fetch.spec.whatwg.org.conf

```diff
@@ -0,0 +1,12 @@
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name fetch.spec.whatwg.org;
+    root /var/www/fetch.spec.whatwg.org;
+
+    ssl_certificate /etc/letsencrypt/live/fetch.spec.whatwg.org/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/fetch.spec.whatwg.org/privkey.pem;
+
+    include /etc/nginx/whatwg.conf;
```

Comment:
https://fetch.spec.whatwg.org/commit-snapshots/ is now 403 forbidden but wasn't before. @domenic, does it matter?

Reply:
I browse that directory index pretty often for the various specs; it would be nice to have one if nginx has that feature.

Reply:
@sideshowbarker, maybe a whatwg-spec.conf that does the extra things?

Reply:
Yeah, we should have a directory listing of sorts.

Reply:
>> https://fetch.spec.whatwg.org/commit-snapshots/ is now 403 forbidden but wasn't before. @domenic, does it matter?
>
> I browse that directory index pretty often for the various specs; it would be nice to have one if nginx has that feature

OK, fixed that and now all `/commit-snapshots` paths are browsable.